packages/web/config/ibc-assets.ts

```diff
@@ -945,12 +945,19 @@ export const IBCAssetInfos: (IBCAsset & {
     counterpartyChainId: "kaiyo-1",
     sourceChannelId: "channel-259",
     destChannelId: "channel-3",
-    coinMinimalDenom: "factory:kujira1qk00h5atutpsv900x202pxx42npjr9thg58dnqpa72f2p7m2luase444a7:uusk",
+    coinMinimalDenom:
+      "factory:kujira1qk00h5atutpsv900x202pxx42npjr9thg58dnqpa72f2p7m2luase444a7:uusk",
     depositUrlOverride:
       "https://blue.kujira.app/ibc?destination=osmosis-1&source=kaiyo-1&denom=factory%2Fkujira1qk00h5atutpsv900x202pxx42npjr9thg58dnqpa72f2p7m2luase444a7%2Fuusk",
     withdrawUrlOverride:
       "https://blue.kujira.app/ibc?destination=kaiyo-1&source=osmosis-1&denom=ibc%2F44492EAB24B72E3FB59B9FA619A22337FB74F95D8808FE6BC78CC0E6C18DC2EC",
   },
+  {
+    counterpartyChainId: "FUND-MainNet-2",
+    sourceChannelId: "channel-382",
```

Comment:
can we please register this IBC connection to the Cosmos Chain Registry? (https://github.com/cosmos/chain-registry/tree/master/_IBC)

Reply:
I made a pull request:
https://github.com/cosmos/chain-registry/pull/914